app/javascript/member-facing/backbone/petition.js

```diff
@@ -17,6 +17,13 @@ const Petition = Backbone.View.extend({
     this.followUpUrl = options.followUpUrl;
     this.submissionCallback = options.submissionCallback;
     this.skipOnSuccessAction = options.skipOnSuccessAction;
+
+    if (options.redirectAfterAction !== undefined) {
+      this.redirectAfterAction = options.redirectAfterAction;
+    } else {
+      this.redirectAfterAction = true;
+    }
+
```

Comment:
I need to disable the redirect since the new template scrolls down after signing a petition (there's no redirect at that time).